cou/utils/openstack.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 """Lookup utils to determine compatible OpenStack codenames for a given component."""
+from __future__ import annotations
```

Comment:
Why is this needed?

Reply:
As suggested by Gabriel in [this comment](https://github.com/canonical/charmed-openstack-upgrader/pull/120#discussion_r1363833076), this is needed for referring the current class in type annotation.